src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -124,21 +133,37 @@ public boolean isBottomLimitSwitchReached() {
     return m_bottomLimitSwitch.get() != ElevatorConstants.kBottomLimitSwitchNC;
   }
 
+  /**
+   * Cap the peak motor power that the talonFX will send to the falcon motor (forward and reverse). 
+   * @param power
+   * @return none
+   */
   public void setMaxOutput(double power) {
     m_motor.configPeakOutputForward(power);
     m_motor.configPeakOutputReverse(power);
   }
 
+  /**
+   * Using the elevator position/extension calculate the actual height of the elevator
+   * from the ground to the bottom of the carriage
+   * 
+   * @return elevator height from ground to bottom of carriage
+   */
   public double getHeight() {
     return Conversions.ElevatorExtensionToHeight(getPosition());
   }
 
+  /**
+   * Set the desired elevator height from the ground. Do this by converting the elevator height to the
+   * required position/extension, then passing in the value to the setDesiredPosition function (Defined above)
+   * @param desiredHeight the height we want the elevator to move to from the ground
+   */
   public void setDesiredHeight(double desiredHeight) {
     setDesiredPosition(Conversions.ElevatorHeightToExtension(desiredHeight));
   }
 
   /**
-   * Get position of carriage above bottom position
+   * Get position of carriage above bottom position. AKA extension
```

Comment:
Would you maybe want to change all references to "position" to instead say "extension"? I think it would be more clear